src/main/scala/ch/epfl/scala/decoder/StackTraceFormatter.scala

```diff
@@ -107,7 +107,8 @@ class StackTraceFormatter(using ThrowOrWarn):
     variable match
       case variable: DecodedVariable.LocalVariable => formatName(variable.symbol)
```

Comment:
`DecodedVariable.LocalVariable` is probably not a the best name. I suggest we rename it to `DecodedVariable.ValDef`. It would be more consistent with `DecodedMethod.ValOrDefDef` and `DecodedField.ValDef`